raiden_contracts/tests/fixtures/channel.py

```diff
@@ -674,11 +676,46 @@ def get(
         _token_network = other_token_network or token_network
         private_key = get_private_key(participant)
 
-        non_closing_signature = sign_balance_proof_update_message(
+        non_closing_signature = sign_balance_proof_message(
             private_key,
             _token_network.address,
             int(_token_network.functions.chain_id().call()),
             channel_identifier,
+            MessageTypeId.BALANCE_PROOF,
+            balance_hash,
+            nonce,
+            additional_hash,
+            original_signature,
+            v,
+        )
+        return non_closing_signature
+
+    return get
+
+
+@pytest.fixture(scope="session")
+def create_balance_proof_updating_countersignature(
+    token_network: Contract, get_private_key: Callable
+) -> Callable:
+    def get(
+        participant: HexAddress,
+        channel_identifier: int,
```

Comment:
This function can still get `msg_type` as a new argument, and one of the similar two functions can be removed.